zerver/models.py

```diff
@@ -797,6 +797,18 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
     bot_type = models.PositiveSmallIntegerField(null=True, db_index=True)  # type: Optional[int]
     bot_owner = models.ForeignKey('self', null=True, on_delete=models.SET_NULL)  # type: Optional[UserProfile]
 
+    # The role field is an eventual replacement for is_realm_admin and is_guest.
+    # See https://github.com/zulip/zulip/issues/13135
+    REALM_ADMINISTRATOR = 120
+    MEMBER = 140
+    GUEST = 160
+    BOT = 300
```

Comment:
Is there reasoning behind this particular set of numbers?  Seem oddly spaced.

Reply:
yeah, I'm not totally sure about this. The reasoning was:
* The first digit is human or bot. 
* Last two digits: We're likely going to fill in roles over time; e.g. primary admin and single-stream guest. The current plan for the human roles is that each role is a strict superset of the previous role. So you could imagine something like `def requires_member(user): return user.role <= UserProfile.MEMBER`.
So I wanted to space the human roles out to allow for future roles to be inserted.

But maybe this is more confusing than it's worth?